packages/tools/src/tools/ZoomTool.ts

```diff
@@ -172,42 +185,31 @@ class ZoomTool extends BaseTool {
         vec3.create(),
         position,
         this.dirVec,
-        -distanceToCanvasCenter * k
+        -distanceToCanvasCenter * zoomFactor
       ) as Types.Point3;
 
       focalPointToSet = vec3.scaleAndAdd(
         vec3.create(),
         focalPoint,
         this.dirVec,
-        -distanceToCanvasCenter * k
+        -distanceToCanvasCenter * zoomFactor
       ) as Types.Point3;
     }
 
-    // If it is a regular GPU accelerated viewport, then parallel scale
-    // has a physical meaning and we can use that to determine the threshold
-    // Added spacing preset in case there is no imageData on viewport
-    const imageData = viewport.getImageData();
-    let spacing = [1, 1, 1];
-    if (imageData) {
-      spacing = imageData.spacing;
-    }
-
     const { minZoomScale, maxZoomScale } = this.configuration;
 
-    const t = element.clientHeight * spacing[1] * 0.5;
-    const scale = t / parallelScaleToSet;
+    const initialScale = this.initialParallelScales.get(viewport.id) || 1;
+    const scale = initialScale / parallelScaleToSet;
 
```

Comment:
I'm not really confident about these changes, especially removing the `imageData` and spacing above. This pull request will take me some time to review and assess across different viewport types (WSI, video, CPU, GPU, 3D) since the changes are pretty significant. So sorry for the delay in advance

Reply:
Yes, let's make it two pull
request

Reply:
Done